Classes/Console/Command/Documentation/GenerateXsdCommand.php

```diff
@@ -0,0 +1,125 @@
+<?php
+declare(strict_types=1);
+namespace Helhum\Typo3Console\Command\Documentation;
+
+/*
+ * This file is part of the TYPO3 Console project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read
+ * LICENSE file that was distributed with this source code.
+ *
+ */
+
+use Helhum\Typo3Console\Command\AbstractConvertedCommand;
+use Helhum\Typo3Console\Service;
+use Helhum\Typo3Console\Service\XsdGenerator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class GenerateXsdCommand extends AbstractConvertedCommand
+{
+    /**
+     * @var XsdGenerator
+     */
+    protected $xsdGenerator;
+
+    public function __construct(string $name = null, XsdGenerator $xsdGenerator= null)
```

Comment:
There's a space missing before the last `=`.

Reply:
done